src/endpoints/verification/verification.service.ts

```diff
@@ -49,4 +51,15 @@ export class VerificationService {
     };
     return await this.rewardService.insert(dataInput);
   }
+
+  async verificationGeneticAnalyst(accountId: string, verificationStatus: string) {
+    await updateGeneticAnalystVerificationStatus(
+      this.subtrateService.api,
+      this.subtrateService.pair,
+      accountId,
+      <GeneticAnalystsVerificationStatus>verificationStatus,
```

Comment:
Why is there a `<` here?

Reply:
for verification_status same with GeneticAnalystsVerificationStatus model...